modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -387,6 +403,15 @@ private void updateLeaseBatchInternal() {
                     return;
                 }
 
+                LeaseUpdateStatisticsParameters totalLeases = leaseUpdateStatistics.onSuccessfulIteration(leasesUpdatedInCurrentIteration);
+
+                if (shouldLogLeaseStatistics) {
+                    LOG.info("Leases updated (printed once per " + LeaseUpdateStatistics.PRINT_ONCE_PER_ITERATIONS + " iterations): ["
+                            + "currentIteration=" + leasesUpdatedInCurrentIteration
+                            + ", total=" + totalLeases
+                            + ", currentAssignmentsSize=" + currentAssignmentsSize + "]");
+                }
```

Comment:
I'd rather insert this printinc logik into the lease statistic object.
Also, this statistic should have functionality to disable it.

Reply:
added system property: LEASE_STATISTICS_PRINT_ONCE_PER_ITERATIONS